### PR TITLE
Add credentials to fetcher IConnection #411

### PR DESCRIPTION
**Pull request by SeungjunWe**
# Description
- Edit a suggestion #412

Before submitting a Pull Request, please test your code. 

If you created a new created a new feature, then create the unit test function, too.

```bash
# COMPILE THE BACKEND SERVER
npm run build

# RUN THE TEST AUTOMATION PROGRAM
npm run test
```

Learn more about the [CONTRIBUTING](CONTRIBUTING.md)

### CODE REVIEW

File: packages/fetcher/src/IConnection.ts [hunk 1]
```diff
@@ -47,4 +47,9 @@ export interface IConnection {
      * @default false
      */
     simulate?: boolean | Partial<IRandomGenerator>;
-}
+
+    /**
+     * A string indicating whether credentials will be sent with the request always, never, or only when sent to a same-origin URL. Sets request's credentials.
+     */
+    credentials?: RequestCredentials;
```
Comment (samchon) [inline packages/fetcher/src/IConnection.ts]: Change to string literal union type please

**Review by samchon — CHANGES_REQUESTED** [review]
Thanks for contribution.

By the way, in NodeJS environment, the `RequestCredentials` type does no exist, so it would be compilation error. Therefore, you'd better to change it as string literal union type instead.

File: packages/fetcher/src/IConnection.ts (hunk 1 above)
Comment (SeungjunWe) [reply packages/fetcher/src/IConnection.ts]: I replace it to string union type

**Review by samchon — APPROVED** [review]
Good, your PR will be published after testing

**Pull request merged**